src/main/resources/modules/CMS130v10.json

```diff
@@ -0,0 +1,281 @@
+{
+  "name": "CMS130",
+  "remarks": [
+    "Synthea module based on CMS130v10 - Colorectal Cancer Screening"
+  ],
+  "states": {
+    "Initial": {
+      "type": "Initial",
+      "direct_transition": "IPP_Age_Guard",
+      "name": "Initial"
+    },
+    "IPP_Age_Guard": {
+      "type": "Guard",
+      "allow": {
+        "condition_type": "And",
+        "conditions": [
+          {
+            "condition_type": "Age",
+            "operator": ">=",
+            "quantity": 50,
+            "unit": "years",
+            "value": 0
+          },
+          {
+            "condition_type": "Age",
+            "operator": "<=",
+            "quantity": 75,
+            "unit": "years",
+            "value": 0
+          }
+        ]
+      },
+      "direct_transition": "Measurement_Period_Guard",
+      "name": "IPP_Age_Guard"
+    },
+    "Terminal": {
+      "type": "Terminal",
+      "name": "Terminal"
+    },
+    "Measurement_Period_Guard": {
+      "type": "Guard",
+      "allow": {
+        "condition_type": "Date",
+        "operator": ">=",
+        "year": 2022,
+        "value": 0
+      },
+      "name": "Measurement_Period_Guard",
+      "distributed_transition": [
+        {
+          "transition": "Telehealth_Visit",
+          "distribution": 0.2
+        },
+        {
+          "transition": "Qualifying_Encounter",
+          "distribution": 0.7
+        },
+        {
+          "transition": "Check_For_Procedure",
+          "distribution": 0.1
+        }
+      ]
+    },
+    "Qualifying_Encounter": {
+      "type": "Encounter",
+      "telemedicine_possibility": "none",
+      "name": "Qualifying_Encounter",
+      "direct_transition": "Encounter_Occurred",
+      "codes": [
+        {
```

Comment:
The `codes` list is a little misleading because only the first code is generally meaningful. In some resource types you might see all of the codes in the output but for Encounters and Procedures as used in this module only the first is relevant. The value_set field means a different code/display may be picked but still only the first entry in the list will show up in the output FHIR. Unfortunately if you want to exercise all these codes you'll either need an overarching value set (probably not possible) or to create separate states for all of them.